Source/ChocolateyGui/Properties/Resources.nb.resx

```diff
@@ -0,0 +1,553 @@
+    <value>Sertifikatpassord</value>
+  </data>
+  <data name="SettingsView_SourcesId" xml:space="preserve">
+    <value>Id</value>
+  </data>
+  <data name="SettingsView_SourcesPassword" xml:space="preserve">
+    <value>Passord</value>
+  </data>
+  <data name="SettingsView_SourcesPriority" xml:space="preserve">
+    <value>Prioritet</value>
+  </data>
+  <data name="SettingsView_SourcesSource" xml:space="preserve">
+    <value>Kilde</value>
+  </data>
+  <data name="SettingsView_SourcesUsername" xml:space="preserve">
+    <value>Brukernavn</value>
+  </data>
+  <data name="SettingsView_ToggleShowConsoleOutput" xml:space="preserve">
+    <value>Vis konsolltekst</value>
+  </data>
+  <data name="ShellView_ButtonAbout" xml:space="preserve">
+    <value>om</value>
+  </data>
+  <data name="ShellView_ButtonSettings" xml:space="preserve">
+    <value>innstillinger</value>
+  </data>
+  <data name="SettingsView_SourcesPath" xml:space="preserve">
+    <value>Sti</value>
+  </data>
+  <data name="LocalSourceView_CheckboxMatchExact" xml:space="preserve">
+    <value>Treff på eksakte ord</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxMatchExactly" xml:space="preserve">
+    <value>Treff på eksakte ord</value>
+  </data>
+  <data name="SettingsView_ButtonRemove" xml:space="preserve">
+    <value>Fjern</value>
+  </data>
+  <data name="SettingsView_SourcesIsDisabled" xml:space="preserve">
+    <value>Er deaktivert</value>
+  </data>
+  <data name="SettingsView_SourcesDisabled" xml:space="preserve">
+    <value>Deaktivert</value>
+  </data>
+  <data name="Controls_PackagesContextMenuInstall" xml:space="preserve">
+    <value>Installer</value>
+  </data>
+  <data name="Controls_PackagesContextMenuPin" xml:space="preserve">
+    <value>Fest</value>
+  </data>
+  <data name="Controls_PackagesContextMenuReinstall" xml:space="preserve">
+    <value>Reinstaller</value>
+  </data>
+  <data name="Controls_PackagesContextMenuUninstall" xml:space="preserve">
+    <value>Avinstaller</value>
+  </data>
+  <data name="Controls_PackagesContextMenuUnpin" xml:space="preserve">
+    <value>Løsne</value>
+  </data>
+  <data name="Controls_PackagesContextMenuUpdate" xml:space="preserve">
+    <value>Oppdater</value>
+  </data>
+  <data name="Controls_PackagesContextMenuDetails" xml:space="preserve">
+    <value>Detaljer</value>
+  </data>
+  <data name="PackageView_ButtonPin" xml:space="preserve">
+    <value>Fest</value>
+  </data>
+  <data name="Bootstrapper_UnhandledException" xml:space="preserve">
+    <value>Ubehandled Unntak</value>
+  </data>
+  <data name="ChocolateyCustomSchemeProvider_InvalidHtml" xml:space="preserve">
+    <value>&lt;h3&gt;Ugyldig HTML Streng&lt;/h3&gt;</value>
+  </data>
+  <data name="ChocolateyCustomSchemeProvider_UnknownHost" xml:space="preserve">
+    <value>&lt;h1&gt;Ukjent Vert&lt;/h1&gt;</value>
+  </data>
+  <data name="ChocolateyRemotePackageService_Escalating" xml:space="preserve">
+    <value>Eskalering</value>

```

Comment:
Helt rett, skal skifte till Eskalerer. Sto helt stille for meg når jeg oversatte denne.